config/secrets.yml

```diff
@@ -29,4 +29,4 @@ test:
 # and move the `production:` environment over there.
 
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 8a0263da45b2a406f4a4fcadb0c9f5c0869a0e1fd101d247572ff774ad93da8bd369ce060ede3c8fa1175e643f91efb7af5759603242223b880db77f7a5bd292
```

Comment:
Whats happening here?

Reply:
I did this to test in production pushed by mistake 🤦‍♂️